vms/vms/settings.py

```diff
@@ -118,6 +118,12 @@
 
 STATIC_ROOT = './static/'
 
+#Instead of sending out real email, during development the emails will be sent
+# to stdout, where from they can be inspected.
+EMAIL_HOST = 'localhost'
+EMAIL_PORT = 1025
```

Comment:
let's set these using os.getenv, it will be easier later. Also port would change to 1025, 587 or 25 depending on the server.